pom.xml

```diff
@@ -5,25 +5,29 @@
     <parent>
         <groupId>edu.unc.lib.cdr</groupId>
         <artifactId>cdr</artifactId>
-        <version>5.0-SNAPSHOT</version>
+        <version>6.0-SNAPSHOT</version>
     </parent>
     <groupId>edu.unc.lib.boxc</groupId>
     <artifactId>cdm-to-boxc-migration-util</artifactId>
     <version>0.0.1-SNAPSHOT</version>
     <name>CDM to Box-c Migration Utility</name>
     <description>Utilities for migrating content from CDM to Box-c</description>
     <properties>
-        <picocli.version>4.6.1</picocli.version>
-        <wiremock.version>2.32.0</wiremock.version>
-        <sqlite-jdbc.version>3.36.0.1</sqlite-jdbc.version>
-        <poi.version>5.1.0</poi.version>
-        <log4j-to-slf4j.version>2.17.1</log4j-to-slf4j.version>
+        <project.java.source>21</project.java.source>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+
+        <compiler.plugin.version>3.14.0</compiler.plugin.version>
+        <picocli.version>4.7.7</picocli.version>
+        <wiremock.version>2.35.2</wiremock.version>
+        <sqlite-jdbc.version>3.50.3.0</sqlite-jdbc.version>
+        <poi.version>5.4.1</poi.version>
+        <log4j-to-slf4j.version>2.18.0</log4j-to-slf4j.version>
         <build-tools.path></build-tools.path>
-        <mysql.version>8.0.28</mysql.version>
-        <csv.version>1.9.0</csv.version>
-        <junit.jupiter.version>5.9.2</junit.jupiter.version>
-        <solr.solrj.version>8.10.1</solr.solrj.version>
-        <embedded-redis.version>0.7.3</embedded-redis.version>
+        <mysql.version>8.0.33</mysql.version>
+        <csv.version>1.13.0</csv.version>
+        <junit.jupiter.version>5.12.1</junit.jupiter.version>
+        <solr.solrj.version>9.8.0</solr.solrj.version>
+        <apache.version>2.15.0</apache.version>
```

Comment:
Should name this `apache-sshd.version` since apache usually refers to an HTTP server